constants/store.js

```diff
@@ -0,0 +1,126 @@
+export default [
```

Comment:
Why is this file not inside a folder like the other pages?
And why is it named "store" here, but "shop" in other places?

Reply:
Shop has only one constant file while other pages have at least two. But naming is definitely wrong